### PR TITLE
Implement callback for Keras

### PR DESCRIPTION
**Pull request by convoliution**
This callback replicates the same hyperparam+observation logging coverage as the manual logging in our [Keras (TensorFlow) example notebook](https://github.com/VertaAI/modeldb-client/blob/development/workflows/examples/tensorflow.ipynb).

Screen Shot 2020-01-16 at 11 18 48 AM

### CODE REVIEW

File: verta/verta/integrations/keras/__init__.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    def set_model(self, model):
+        try:
+            optimizer = model.optimizer._name
+            self.run.log_hyperparameter("optimizer", optimizer)
+        except:
+            pass  # don't halt execution
+
+        try:
+            loss = model.loss
+            self.run.log_hyperparameter("loss", _utils.to_builtin(loss))
```
Comment (conradoverta) [inline verta/verta/integrations/keras/__init__.py]: I'm assuming `to_buildin` will be able to read the class type from `loss`, right?

Comment (convoliution) [reply verta/verta/integrations/keras/__init__.py]: Oh dear it does not; I only tried this when `model.loss` is a string, not one of Keras's loss classes/functions.
